dep/boost/CMakeLists.txt

```
@@ -83,3 +83,23 @@ endif()
 target_compile_definitions(boost
   INTERFACE
     -DTC_HAS_BROKEN_WSTRING_REGEX)
+
+if (WITH_BOOST_STACKTRACE)
+  if (BOOST_STACKTRACE_BACKTRACE_INCLUDE_FILE)
+    CHECK_INCLUDE_FILE(${BOOST_STACKTRACE_BACKTRACE_INCLUDE_FILE} HAS_BACKTRACE)
+  else()
+    CHECK_INCLUDE_FILE("backtrace.h" HAS_BACKTRACE)
+  endif()
+
+  if (NOT HAS_BACKTRACE)
+    message(FATAL_ERROR "Required header 'backtrace.h' not found. If building with a compiler other than GCC please specify the full path in the CMake option BOOST_STACKTRACE_BACKTRACE_INCLUDE_FILE")
```

Punctuation details:
```suggestion
    message(FATAL_ERROR "Required header 'backtrace.h' not found. If building with a compiler other than GCC, please specify the full path in the CMake option BOOST_STACKTRACE_BACKTRACE_INCLUDE_FILE.")
```
(You may skip the ending period dot, if you feel it takes away focus from the message.)